uwblueprint/building-up-client
# Storefront Checkout Cart Design Changes

## Description

A bunch of misc. design changes & updates:

Link to tickets:

- Shopping bag # of items on the icon: https://www.notion.so/uwblueprintexecs/Shopping-Bag-of-items-on-the-icon-38396bad914d4cdca45b9eb87666aa66
- Shopping bag order summary updates: https://www.notion.so/uwblueprintexecs/Shopping-Bag-Order-summary-updates-33c2a2637db84a65b85ba4e80aa0b349
- Shopping bag no items view: https://www.notion.so/uwblueprintexecs/Shopping-Bag-No-items-view-a70efd39e95f49d7a8b0570e8306f962
- Quantity picker smaller size: https://www.notion.so/uwblueprintexecs/Quantity-picker-smaller-size-763a75c1dd6a4a63b6181593ea6f89e6
- Other misc. design changes: https://www.notion.so/uwblueprintexecs/Checkout-Design-Changes-3b8ef35b302640a3bbc1fc73da663f67

## Approach

Mainly styling changes but there's a few small logic changes: (just going to leave comments on specific areas where I have questions)

## Testing

Icon on bag:
- Verify that the numbers accurately reflect what's inside the cart (will not appear if there are 0 items and will be `9+` if there's more than 9)
- Let me know if we want to make the font/circle bigger or smaller

Quantity Picker
- Verify that size is good on the checkout page and the size is a lot bigger on product pages

No items view
- Remove all items from your cart, or directly access the cart when you have nothing in it
- Verify that spacing & everything looks good

Misc. 
- Try to navigate directly to the product page by pressing the product image on the checkout page
- Make sure the total in your order summary is correct, and when you proceed to checkout everything works fine

## Screenshots

Updated: icon on bag, spacing on checkout page, fonts, and order summary
![image](https://user-images.githubusercontent.com/24748963/115792842-fa9ece00-a398-11eb-860b-b8d0a89f724a.png)
![image](https://user-images.githubusercontent.com/24748963/115792902-1904c980-a399-11eb-8eef-be9cc6de5b12.png)
